tox.ini

```
@@ -44,14 +44,28 @@ commands =
 [testenv:unit]
 description = Run unit tests
 deps =
-    pytest==7.2.0
-    coverage[toml]==6.5.0
+    pytest
+    coverage[toml]
     -r{toxinidir}/requirements.txt
 commands =
-    coverage run --source={[vars]src_path} \
-        -m pytest \
-        --tb native \
-        -v \
-        -s \
-        {posargs}
+    coverage run
```

Just caught this - you are missing a line continuation character here. It is causing the unit tests to fail since it thinks there is nothing to do. I did have to fix a merge conflict, so this might have been my fault :cold_face:. Let me commit a new fix quickly.

```suggestion
    coverage run \
```